src/main/java/reposense/system/CustomLogFormatter.java

```diff
@@ -19,7 +20,8 @@
  */
 public class CustomLogFormatter extends SimpleFormatter {
 
-    private static final DateFormat dateFormat = new SimpleDateFormat("hh:mm:ss");
+    private static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("hh:mm:ss")
+            .withZone(ZoneId.systemDefault());
```

Comment:
I am also not sure about what to do with this one.

Reply:
What if you just remove the `.withZone`?